lib/chef/provider/package/rubygems.rb

```diff
@@ -517,7 +517,7 @@ def install_package(name, version)
               install_via_gem_command(name, version)
             end
           elsif @new_resource.gem_binary.nil?
-            @gem_env.install(@new_resource.source)
+            @gem_env.install(@new_resource.source, domain: :local)
```

Comment:
It'd be nice to have a comment here pointing out that `domain` is used by `Gem::DependencyInstaller` rather than by Chef code.